app/src/main/res/xml/progress_scene.xml

```diff
@@ -0,0 +1,84 @@
+<?xml version="1.0" encoding="utf-8"?>
+<MotionScene
+        xmlns:android="http://schemas.android.com/apk/res/android"
+        xmlns:motion="http://schemas.android.com/apk/res-auto"
+        >
+
+    <Transition
+            motion:constraintSetEnd="@+id/end"
+            motion:constraintSetStart="@+id/start"
+            motion:duration="500"
+            />
+
+    <ConstraintSet android:id="@+id/start">
+
+        <Constraint
+                android:id="@+id/noiseSensitivityProgress"
+                android:layout_width="0.1dp"
```

Comment:
Is that some kind of hax? ( ͡° ͜ʖ ͡°)

Reply:
It's for the scene scaling, dunno if there is some nicer way for it. Couldn't use 0dp as it is match_parent in constraint layout ^^